### DISCUSSION TITLE
Unable to use 2fa

### DISCUSSION THREAD

**Discussion by DivyanshMandhan** [community/community#109755] [Other Feature Feedback, Questions, & Ideas] [answered]
Logged out from an account

My TOTP is not working and I don't have recovery codes, any way I can login to my account?

**Comment by ThisIsMrIsmail** [comment, marked as answer]
Hello @DivyanshMandhan,

This is tricky. Without your codes, it's hard to get back in. Here's what to try:

**Contact GitHub Support ASAP:** They might be able to help, but it could take time. Here's the link: https://support.github.com/contact
**Search EVERYWHERE:** Look through old emails, notes, etc., Did you save those recovery codes somewhere without realizing it?
Important:

If you get back in, save backup codes safely!
Set up other ways to log in (like your phone) so this doesn't happen again."

**Reply by DivyanshMandhan** [reply]
I tried with PAT and recovery, 

"You have completed the request to disable two-factor authentication for the account xxx.

GitHub Support will review this request within approximately 1-3 business days."
- from GitHub

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @DivyanshMandhan! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
